north/Cargo.toml

```diff
@@ -17,6 +17,7 @@ async-trait = "0.1.36"
 base64 = "0.12.3"
 bitflags = "1.2.1"
 config = "0.10.1"
+ed25519-dalek = "1.0.0-pre.4"
```

Comment:
is there a non-alpha version also?

Reply:
The 0.9 version seems broken because rand crate has upgraded meanwhile.
Also its API is different than for 1.0(-pre).
Let's just hope they get to final 1.0.0 soon.